public/CNAME

```diff
@@ -1 +1 @@
-new.hnscan.com
+hnscan.com
```

Comment:
Ahh, maybe that's why that other PR was needed to update CNAME, perhaps we simply forgot to change this sucker?

Reply:
Damn, forgot that I had update it here :(.